docs/source/install.rst

```diff
@@ -84,21 +86,12 @@ To build ``libsemigroups_pybind11`` from a release archive:
 Building the documentation
 --------------------------
 
-The following are required to be able to build the documentation:
-
-1. ``python3``
-2. the python packages: ``jinja2 sphinx sphinx_rtd_theme sphinxcontrib-bibtex
-   sphinx_copybutton``
-
-Assuming you already have ``python3`` install, on Mac OSX you can install all of
-the above by doing:
+Assuming you already have ``python3`` install, you can build the doc by using:
 
 ::
 
-    python3 -m pip3 install -r docs/requirements
-
-Then it ought to be possible to just run ``make doc`` in the ``libsemigroups``
-directory.
+    python3 -m pip3 install -r requirements.txt
+    make doc
```

Comment:
Do we expect all of our target machines to have some form of `automake` / a tool that can execute `make doc`? If yes, this is fine. If no, this could be replaced with
``` bash
./etc/make-doc.sh
```